tests/helper.py

```diff
@@ -28,6 +33,35 @@ def __init__(self):
         if any('admin' in s for s in token.get_data()['permissions']):
             raise Exception('The test is run with an admin user account. Cancelled, please try again with a normal account!')
 
+    @classmethod
+    def load_env(cls):
+        env_file_name = '.env'
+        default_env_keys = set(['email', 'user_id', 'password'])
+        cls.credentials = {}
+
+        if Path.isfile(env_file_name):
+            # If .env file exists
+            config_reader = configparser.ConfigParser()
+            config_reader.read(env_file_name)
+            config_data = {}
+            options = config_reader.options('Credentials')
+            for option in options:
+                try:
+                    config_data[option] = config_reader.get('Credentials', option)
+                except:
+                    config_data[option] = None
+            if not default_env_keys.issubset(set(config_data)):
+                raise Exception('Mandatory env keys missing!')
+            cls.credentials = config_data
+        else:
+            # If .env file not exists, read credentials from environment vars.
+            try:
+                cls.credentials['email'] = os.environ.get('TEST_ENV_EMAIL')
+                cls.credentials['user_id'] = os.environ.get('TEST_ENV_USER_ID')
+                cls.credentials['password'] = os.environ.get('TEST_ENV_PASSWORD')
+            except:
+                raise Exception('Mandatory env keys missing!')
```

Comment:
This will never be thrown, as `os.environ.get` doesn't throw if the variable doesn't exist (it just returns `None`)

Reply:
@pimterry updated, by doing `os.environ['Key']`, it will raise `KeyError` exception, if the environment variable is not present.